format/run-clang-format.py

```diff
@@ -203,12 +203,19 @@ def print_trouble(prog, message, use_colors):
 
 
 def main():
+
+    with open(os.path.join(os.path.dirname(os.path.dirname(
```

Comment:
Why don't you just import it like we do in `frontend.py`?

Reply:
Because formatting should be doable even before SMACK is installed. If so, we haven't installed `bin/versions` as a Python module.